src/pages/jobs.js

```diff
@@ -0,0 +1,160 @@
+import { useState } from 'react'
+import Form from '../components/Jobs/Form/Form'
+import Card from '../components/Jobs/Card/Card'
+import Loader from '../components/Jobs/Loader/Loader'
+import Video from '../components/Jobs/Video/Video'
+import Pagination from '../components/Pagination/Pagination'
+import PageHeader from '../components/PageHeader'
+import { StaticImage } from 'gatsby-plugin-image'
+import SEO from '../components/SEO'
+
+function Jobs() {
+  const [jobData, setJobData] = useState(false)
+  const [formSubmitEvent, setFormSubmitEvent] = useState(false)
+  const [formSubmitted, setFormSubmitted] = useState(false)
+  const [pageContext, setPageContext] = useState(false)
+
+  function formData(formResponse, page = 1) {
+    setFormSubmitEvent(formResponse)
+    setFormSubmitted(true)
+    document.getElementById('scroll-to').scrollIntoView({ behavior: 'smooth', block: 'start' })
+
+    let what = 'javascript Reactjs gatsby graphql nodejs node.js jquery bootstrap'
+    let exclude = '0000 senior sr principal lead master'
+
+    let url = `https://8oi4im0qo3.execute-api.us-east-1.amazonaws.com/dev/jobsearch/${page}?&results_per_page=15&what_or=${what}&where=${
+      formResponse.zipCode
+    }&distance=${formResponse.distance}&what_exclude=${exclude}&sort_by=date&max_days_old=30${
+      formResponse.remote === true ? '&what_and=remote' : ''
+    }`
+
+    fetch(url)
+      .then(response => response.json())
+      .then(data => {
+        setJobData(data)
+        setPageContext({
+          currentPage: page,
+          minPage: Math.floor(page / 10) * 10,
+          maxPage: Math.ceil(page / 10) * 10 - 1,
+          totalPages: Math.floor(data.count / 15),
+          formResponse: formResponse,
+          formData: formData,
+          setPageContext: setPageContext,
+        })
+        console.log(pageContext)
```

Comment:
Is there a reason for the `console.log()`?

Reply:
removed